templates/terraform/resource_definition/ssl_policy.erb

```diff
@@ -0,0 +1,34 @@
+<% if false # the license inside this if block pertains to this file -%>
+	# Copyright 2017 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+<% end -%>
+CustomizeDiff: func(diff *schema.ResourceDiff, v interface{}) error {
```

Comment:
My mild preference is to stick this function in `constants`, then refer to it by name in `CustomizeDiff`.

Reply:
Done